samples/tls_between_host_enclave/server/enc/crypto.cpp

```diff
@@ -0,0 +1,313 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "crypto.h"
```

Comment:
Can the enclave code be shared between the two samples?

Reply:
Good point.
These two samples are targeting at different audiences. 
I would prefer each sample independent from each other. I will think of way to see if I could make it more interesting on the 2nd copy by adding different features to it.